### PR TITLE
WIP: Text-Utilities (Typography)

### PR DESCRIPTION
**Pull request by way2nizam**
Screenshot from 2022-02-20 19-44-25

### CODE REVIEW

File: text-utilities/index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+        <h3>This is a sample text.</h3>
+
+        <h4>This is a sample text.</h4>
+
+        <h5>This is a sample text.</h5>
+
+        <h6>This is a sample text.</h6>
+
+
```
Comment (chanchal-panpaliya) [inline text-utilities/index.html]: remove extra space.

**Review by chanchal-panpaliya — APPROVED** [review]
looks good to merge

**Review by KanekarSnehal — APPROVED** [review]
No issues found, you can merge it.

File: text-utilities/index.html (hunk 1 above)
Comment (way2nizam) [reply text-utilities/index.html]: done, thank you chanchal.

**Pull request merged**